### ISSUE TITLE
render not respecting output_file when run_pandoc = F

### ISSUE THREAD

**Issue by psads-git** [rstudio/rmarkdown#2395]
Take the following command:

`rmarkdown::render("parsing_r_questions.Rmd", output_format = "md_document", output_file = "parse.md", run_pandoc = F)
`

The output file will be 

`parsing_r_questions.knit.md`

and not 

`parse.md`.

**Comment by cderv** [comment]
Yes `output_file` is used only for the output of Pandoc conversion really. `run_pandoc = FALSE` is a way to keep the intermediate of conversion, the md after knitting and other related information. It won't be the full `md_document()` format - only the **knitr** related part of the output format definition will be used. 

Why do you need `run_pandoc = FALSE` when converting to `md_document` output format ? 

However, you could try adding `keep_md: TRUE` in your format definition. The `parsing_r_questions.knit.md` should be renamed with `output_file` value instead - but it still won't be the full result of `md_document()` output format, it will only be the `md` file resulting of the knitting step

**Comment by psads-git** [comment]
Thanks, Christophe. I was trying to study the differences between markdown files generated using and not using pandoc, when I noticed that `output_file` was not being respected when `run_pandoc = F`. I thought it could maybe be a bug, and I therefore reported it.
